packages/lib-user/src/components/UserHome/components/RecentSubjects/RecentSubjects.js

```diff
@@ -0,0 +1,68 @@
+import { Anchor, Box, ResponsiveContext, Text } from 'grommet'
+import { arrayOf, bool, shape, string } from 'prop-types'
+import { useContext } from 'react'
+import { Loader, SpacedHeading, SpacedText } from '@zooniverse/react-components'
+
+import SubjectCard from '../SubjectCard/SubjectCard.js'
+
+function RecentSubjects({ isLoading = false, subjects = [] }) {
+  const size = useContext(ResponsiveContext)
+
+  return (
+    <Box
+      pad='medium'
+      height={{ min: '200px' }}
+      round='small'
+      border={{
+        color: { light: 'light-5', dark: 'black' },
+        size: 'xsmall'
+      }}
+    >
+      <SpacedHeading size='1.125rem' level={2} margin={{ top: '0' }}>
+        Recent Classifications
+      </SpacedHeading>
+      <Box
+        as='ul'
+        direction='row'
+        gap='small'
+        pad={{ horizontal: 'xxsmall', bottom: 'xsmall' }}
+        overflow={{ horizontal: 'auto' }}
+        style={{ listStyle: 'none' }}
+        margin='0'
+      >
+        {isLoading && (
+          <Box fill justify='center' align='center'>
+            <Loader />
+          </Box>
+        )}
```

Comment:
[ui] Loaders = good! 👍 I always appreciate status feedback for users.

[ui][future improvement] We should also figure out a UI standard for showing error messages, should something go sideways.

Reply:
Good point! I'll add a basic error message to this component for now.